recipe/install_nvcc.sh

```diff
@@ -91,18 +91,22 @@ export CMAKE_ARGS="\${CMAKE_ARGS}"
 ### /CMake configurations
 
 mkdir -p "\${CONDA_BUILD_SYSROOT}/lib"
+mkdir -p "\${CONDA_PREFIX}/lib/stubs"
 
 # Add \$(libcuda.so) shared object stub to the compiler sysroot.
 # Needed for things that want to link to \$(libcuda.so).
 # Stub is used to avoid getting driver code linked into binaries.
 
-# Make a backup of \$(libcuda.so) if it exists
-if [[ -f "\${CONDA_BUILD_SYSROOT}/lib/libcuda.so" ]]
+
+if [[ "\${CONDA_BUILD_SYSROOT}" != "" ]]
```

Comment:
Should catch whether it is defined

```suggestion
if [[ ! -z "\${CONDA_BUILD_SYSROOT+x}" ]]
```